modules/mauth-signer/src/main/java/com/mdsol/mauth/Signer.java

```diff
@@ -18,6 +18,23 @@
    * @return MAuth headers which should be appended to the request before sending.
    * @throws MAuthSigningException when request cannot be signed
    */
-  Map<String, String> generateRequestHeaders(String httpVerb, String requestPath,
-                                             String requestPayload) throws MAuthSigningException;
+  Map<String, String> generateRequestHeaders(String httpVerb,
```

Comment:
Again, I would use `generateRequestHeaders` for the desired default of v2 and rename this and deprecate

Reply:
It may cause some issue with the existing modules which is calling **generateRequestHeaders** if rename it.  We need to think a better name for generateRequestHeadersV2, how about generateRequestHeadersEx?

Reply:
generateRequestHeaders should be th most suitable name, renamed the both method name as suggested. See  https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/24c95e0625d79457b80781f3972c4f7f2f1e9021